src/main/java/plugily/projects/villagedefense/arena/managers/maprestorer/MapRestorerManager.java

```diff
@@ -139,10 +142,10 @@ public void restoreTopHalfDoorPart(Block block) {
     doorBlockState.update(true);
   }
 
-  public void restoreBottomHalfDoorPart(Block block, byte doorData) {
+  public void restoreBottomHalfDoorPart(Block block) {
     block.setType(Utils.getCachedDoor(block));
     BlockState doorBlockState = block.getState();
-    Door doorBlockData = new Door(TreeSpecies.GENERIC, arena.getPlugin().getBukkitHelper().getFacingByByte(doorData));
+    Door doorBlockData = new Door(TreeSpecies.GENERIC, arena.getPlugin().getBukkitHelper().getFacingByByte((byte) 1));
```

Comment:
does the facing works always?

Reply:
it adjusts automagically, might require further research if will work under all conditions, but did work at test server, just messed up front with back which is a minor issue that can be ignored